frontend/src/modules/projectinfo/components/ProjectInfo.js

```diff
@@ -0,0 +1,55 @@
+/* @flow */
+
+import * as React from 'react';
+import { connect } from 'react-redux';
+
+import './ProjectInfo.css';
+
+import * as navigation from 'core/navigation';
+import * as project from 'core/project';
+
+import InfoPanel from './InfoPanel';
+
+import type { NavigationParams } from 'core/navigation';
+import type { ProjectState } from 'core/project';
+
+
+type Props = {|
+    parameters: NavigationParams,
+    project: ProjectState,
+|};
+
+type InternalProps = {|
+    ...Props,
+    dispatch: Function,
+|};
+
+
+/**
+ * Render the current project's completion status and its info panel.
+ */
+export class ProjectInfoBase extends React.Component<InternalProps> {
+    render() {
+        if (
+            this.props.project.fetching ||
+            !this.props.project.name ||
+            this.props.parameters.project === 'all-projects'
+        ) {
+            return null;
+        }
+
+        return <div className="project-info">
```

Comment:
We should be able to remove this `<div>` and join this component with `InfoPanel`.

ProgressBar should be a separate component and the two don't need a dedicated wrapper component. Semantically, they are different. Note that the ProgressBar represents the Resource progress, not Project progres.

Reply:
Ah, I didn't know the progress bar was about the resource. I guess when we're on "all-resources" it still shows the project's data, though?

Reply:
Yes, in this case it is. It's just a different and more visible presentation of numbers in the filter, that's why we'd like to join the two in the future somehow.